v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,311 @@
+
+See [`CREATE CHANGEFEED`](create-changefeed.html) for examples on using the foundational syntax to create a changefeed. For information on sinks, see the [Changefeed Sinks](changefeed-sinks.html) page.
+
+### Filter columns
+
+To only emit data from specific columns in a table, you can use `SELECT {columns}` to define the table's columns.
+
+As an example, using the `users` table from the [`movr` database](movr.html#the-movr-database), you can create a changefeed that will emit messages including only the `name` and `city` column data:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+CREATE CHANGEFEED INTO "scheme://sink-URI" WITH updated, schema_change_policy = 'stop' AS SELECT name, city FROM users;
+~~~
+~~~
+{"record":{"users":{"name":{"string":"Steven Lara"},"city":{"string":"los angeles"}}}}
+{"record":{"users":{"city":{"string":"los angeles"},"name":{"string":"Carl Russell"}}}}
+{"record":{"users":{"name":{"string":"Brett Porter"},"city":{"string":"boston"}}}}
+{"record":{"users":{"name":{"string":"Vanessa Rivera"},"city":{"string":"los angeles"}}}}
+{"record":{"users":{"name":{"string":"Tony Henderson"},"city":{"string":"los angeles"}}}}
+{"record":{"users":{"city":{"string":"boston"},"name":{"string":"Emily Hill"}}}}
+{"record":{"users":{"name":{"string":"Dustin Kramer"},"city":{"string":"boston"}}}}
+{"record":{"users":{"name":{"string":"Dawn Roman"},"city":{"string":"boston"}}}}
+~~~
+
+### Filter delete messages
+
+To remove the [delete messages](changefeed-messages.html#delete-messages) from a changefeed stream, use the [`cdc_is_delete()`](#cdc-transformation-function-support) function:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED INTO sink WITH schema_change_policy = 'stop' AS SELECT * FROM table WHERE NOT cdc_is_delete();
+~~~
+
+Filtering delete messages from your changefeed is helpful for certain outbox table use cases. See [Transformations and the outbox pattern](#transformations-and-the-outbox-pattern) for further detail.
+
+### Geofilter a changefeed
+
+When you are working with a [`REGIONAL BY ROW` table](set-locality.html#regional-by-row), you can filter the changefeed on the `crdb_region` column to create a region-specific changefeed:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE CHANGEFEED INTO sink WITH schema_change_policy = 'stop' AS SELECT * FROM table WHERE crdb_region = 'europe-west2';
+~~~
+
+For more detail on targeting `REGIONAL BY ROW` tables with changefeeds, see [Changefeeds in Multi-Region Deployments](changefeeds-in-multi-region-deployments.html).
+
+### Stabilize the changefeed message schema
+
+As changefeed messages emit from the database, message formats can vary as tables experience [schema changes](changefeed-messages.html#schema-changes). You can select columns with typecasting to prevent message fields from changing during a changefeed's lifecycle: 
```

Comment:
nit: worth linking `typecasting` here: https://www.cockroachlabs.com/docs/stable/data-types.html#data-type-conversions-and-casts ?